pkg/users/local.js

```diff
@@ -998,6 +998,9 @@ PageAccount.prototype = {
 
         var proc;
         var checked = $(ev.target).prop('checked');
+        var input_elements =
+            $('#account button:not([disabled]), #account input:not([disabled]), #account a:not([disabled])');
```

Comment:
Would it make sense to make this a global variable and set it in `init()`, once? Then you also wouldn't need the `:not([disabled])` qualifier, wouldn't have to replicate this three times, and the XPath query wouldn't need to be re-done on every function call.

To simplify this further, could the whole `<div id="accounts">` be disabled, or does that look odd or not work?

Reply:
> To simplify this further, could the whole `<div id="accounts">` be disabled, or does that look odd or not work?

I tried that, but it didn't seem to work for me. I disabled it the same way, so `$('#accounts').prop('disabled', true)`. But no luck, is it supposed to work that way? Maybe with `.prop('disabled', 'disabled')`? 

> global variable

Maybe, elements get disabled/reenabled dynamically in other ways too though. For instance you have the `Terminate session` button which can get enabled/disabled after doing any action where `self.update()` is called. And this shouldn't interfere with that. If that's the only case this happens however maybe it's worth making the global variable.

Reply:
I see, for these it actually *does* need to re-compute the set of non-disabled action elements. Makes sense, thank you!